tools/rosbag/README.md

```diff
@@ -0,0 +1,32 @@
+# rosbag
+This is a custom rosbag package which provides all the feature of [original rosbag](http://wiki.ros.org/rosbag/Commandline) with custom freq recording of topics.
+
+### feature of package
+* saves storage
+* decreases cpu usage
```

Comment:
It is a bit strange to mention this here. This is an indirect consequence of the feature you added, but not a general feature of the package (this readme referrs to the rosbag package as a whole).
I would just remove this section.